test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -90,7 +90,10 @@ private void assertCommandBehavior(String inputCommand,
         //Confirm the state of data is as expected
         assertEquals(expectedAddressBook, addressBook);
         assertEquals(lastShownList, logic.getLastShownList());
-        assertEquals(addressBook, saveFile.load());
+        // Since not all commands actually mutates the addressbook on storage, it is normal that the addressBook in memory
```

Comment:
Alternatively, you can check if the command is mutating and do `assertEquals(addressBook, saveFile.load())` when it is. If the command is not mutating, you can check if the file was not updated (by checking the last updated time of the file)